addons/project/models/project.py

```diff
@@ -53,6 +53,7 @@
     'dependent_ids',
     'is_blocked',
     'message_is_follower',
+    'date_last_stage_update',
```

Comment:
Why is it needed? 🤔

Reply:
This is given in the write list so now it is not needed in read list.so i removed. [link](https://github.com/odoo/odoo/blob/master/addons/project/models/project_task.py#L51) thanks